src/iminuit/minuit.py

```diff
@@ -2314,13 +2314,19 @@ def interactive(
         self,
         plot: Callable = None,
         raise_on_exception=False,
+        qt_exec=True,
```

Comment:
I don't like this option. I suspect there is a way to find out whether a QApplication is already running, and only start the QApplication and its event loop when it does not already exist. I would prefer that over this option, which is rather technical.

Reply:
I removed it, but not in the tests yet. I'll have to figure out how I can make the tests work later.

Reply:
As mentioned in some another comment, the more I think about it the more ok I am with this, if the user never sees this.